src/modules/problemTypes.ts

```diff
@@ -64,3 +69,5 @@ export type ProblemWithRelationsAndProblemSection = ProblemFindManyWithAuthorAnd
     CheckIsAvailableProblemType;
 
 export type ProblemWithRelations = ExtractPrismaTypeFromArray<typeof problemMethods.getList>;
+
+export type ProblemHistoryWithUser = ProblemHistory & { user: User };
```

Comment:
I'm sure that this should be higher than ProblemFindManyWithAuthorAndTagsAndFavourite